pom.xml

```
@@ -32,15 +32,17 @@
   </scm>
 
   <properties>
-    <revision>6.1.4</revision>
+    <revision>6.2.0</revision>
     <changelist>-SNAPSHOT</changelist>
     <!-- Character set tests fail unless file.encoding is set -->
     <argLine>-Dfile.encoding=${project.build.sourceEncoding}</argLine>
     <gitHubRepo>jenkinsci/${project.artifactId}-plugin</gitHubRepo>
     <!-- https://www.jenkins.io/doc/developer/plugin-development/choosing-jenkins-baseline/ -->
     <jenkins.baseline>2.479</jenkins.baseline>
     <jenkins.version>${jenkins.baseline}.3</jenkins.version>
-    <jgit.version>7.0.0.202409031743-r</jgit.version>
+    <jgit.version>7.2.0.202503040940-r</jgit.version>
```

A newer JGit version is available
```suggestion
    <jgit.version>7.2.1.202505142326-r</jgit.version>
```